language-service/src/services/yamlCompletion.ts

```diff
@@ -404,7 +418,7 @@ export class YAMLCompletion {
                 insertText: this.getInsertTextForValue(value, separatorAfter),
                 insertTextFormat: InsertTextFormat.Snippet,
                 detail: localize('json.suggest.default', 'Default value'),
-            });
+            }, false);
```

Comment:
Do we need to check case here ?

Reply:
I don't think so.  This code path looks like it's only run when node is null and I haven't figured out when, if ever, that happens.  But more generally since it is completing for a property value, there's nothing it could collide with like is possible with property keys.